content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -182,7 +170,7 @@ If the **Request** has a header `User-ID`, its value is applied as tag `http.req
 If the **Response** has a header `User-ID`, its value is applied as tag `http.response.headers.User-ID`.<br><br>
 Added in version 1.18.3.<br>
 Response header support and entries without tag names added in version 1.26.0.<br>
-**Beta**: Starting in version 2.35.0, if [Agent Remote Configuration][16] is enabled where this service runs, you can set `DD_TRACE_HEADER_TAGS` in the [Service Catalog][17] UI. 
+**Beta**: Starting in version 2.35.0, if [Agent Remote Configuration][16] is enabled where this service runs, you can set `DD_TRACE_HEADER_TAGS` in the [Service Catalog][17] UI.
```

Comment:
This has been available for a long time now, shall we remove the Beta label (seeing as we use preview now anyway)

Reply:
Updated in 497eb78cf33d7dc1fb7916a2086e8a9067905ec9 . Thanks for the suggestion!